getting-started/templates/sizing-examples/core-sizing-example.yaml

```diff
@@ -0,0 +1,258 @@
+  autoscaling:
+    minReplicas: 2
+    maxReplicas: 10
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 80
+
+## User Services stores account information and configured permissions for SystemLink users.
+userservices:
+  resources:
+    requests:
+      memory: 350Mi
+      cpu: 250m
+    limits:
+      memory: 350Mi
+  autoscaling:
+    minReplicas: 4
+    ## Increase the replica count to support a larger client load.
+    maxReplicas: 15
+    targetCPUUtilizationPercentage: 70
+    targetMemoryUtilizationPercentage: 80
+
+  ## To improve performance, the user service maintains a cache of API key authorizations. This configuration
+  ## controls the behavior of that cache.
+  authorizationCache:
+    ## @param userservices.authorizationCache.expirationTimeSeconds The number of seconds after the last key access
+    ## when the authorization result will remain in the cache.
+    ## DEFAULT: 5 minutes
+    expirationTimeSeconds: 300
+    ## A subset of the cache will be refreshed in the background at this frequency, prioritizing recently accessed keys.
+    refresh:
+      ## @param userservices.authorizationCache.refresh.frequencySeconds The frequency at which the cache refresh runs.
+      ## Should be smaller than `expirationTimeSeconds`.
+      ## DEFAULT: 1 minute
+      frequencySeconds: 60
+      ## @param userservices.authorizationCache.refresh.batchSize The number of keys updated in each refresh period.
+      ## Increasing this value will increase accuracy of the cache, at
+      ## the cost of increased resource usage.
+      batchSize: 250
+      ## @param userservices.authorizationCache.refresh.maxConcurrency The number of current tasks used to implement the refresh.
+      ## Increasing this value will allow the refresh to run more quickly, at the cost of increased resource usage.
+      maxConcurrency: 5
+
+## The Web Application Service hosts user-defined web applications and stores published notebooks and data space configurations.
+webappservices:
+  resources:
+    requests:
+      cpu: 500m
+      memory: 2Gi
+    limits:
+      memory: 2Gi
+  autoscaling:
+    minReplicas: 2
+    maxReplicas: 10
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 70
+
+  ## This shows an example of configuring an nginx ingress to support large uploads.
+  ingress:
+    annotations:
+      ## Increase the limit on the size of requests sent through the ingress.
+      nginx.ingress.kubernetes.io/proxy-body-size: 100m
+
+  ## @param webappservices.notebookMaximumUploadSizeInMegabytes Maximum size of a notebook that can be uploaded
+  ## using notebook APIs in MB.
+  notebookMaximumUploadSizeInMegabytes: "10"
+
+  ## @param webappservices.notebookMaximumUploadSizeInMegabytes The time between cleanups of web apps in seconds.
+  webAppsCleanUpServiceDelaySeconds: "900"
+  ## @param webappservices.notebookMaximumUploadSizeInMegabytes The time in seconds before a webapp marked for deletion
```

Comment:
Should there be an extra newline above this comment to match the commen+configuration grouping space elsewhere in this doc?

Reply:
done